interface/weno/transmitRx.php

```diff
@@ -0,0 +1,68 @@
+<?php
```

Comment:
take the cap letter out of this file:
interface/weno/transmitrx.php
(should only be using caps in class dir/names)

Reply:
completed